### DISCUSSION TITLE
HINT: Set both the width_field and height_field attribute to avoid storage IO

### DISCUSSION THREAD

**Discussion by LeoGraciano** [codingjoe/django-pictures#64] [Q&A]
Console INFO:
product.Product.thumbnail: (fields.E101) width_field and height_field attributes are missing
        HINT: Set both the width_field and height_field attribute to avoid storage IO

```
    thumbnail = PictureField(
        _('Thumbnail'),
        upload_to="img/thumbnail",
    )
```
try save in django admin... img save more, return exception
![image](https://user-images.githubusercontent.com/21141434/201899173-ed9bd585-e8a0-4175-b1ad-9836e082c4da.png)

**Comment by codingjoe** [comment]
Hi @LeoGraciano,

Thank you for reaching out! To better help you, it would be wonderful if you could share the entire stack trace with me. At first glance, the issues seem to be with your Celery setup. Do you know if Celery is working locally? Maybe your Redis or RabbitAMQP server isn't running?

Cheers,
Joe

**Reply by LeoGraciano** [reply]
I user Celery with Redis…

**Reply by codingjoe** [reply]
Maybe try restarting your local redis server? Based on what I see on your screenshot, the server seems to be down, since kombu can't even establish a TCP connection.
